lib/matplotlib/markers.py

```
@@ -105,16 +112,24 @@ def get_marker(self):
-        elif is_math_text(marker):
-            self._marker = marker
-            self._marker_function = self._get_mathtext_path
+                self, '_set_' + self.markers[marker])
+        elif is_string_like(marker) and is_math_text(marker):
+            self._marker_function = self._set_mathtext_path
+        elif isinstance(marker, Path):
+            self._marker_function = self._set_path_marker
         else:
-            raise ValueError('Unrecognized marker style %s' % marker)
+            try:
+                path = Path(marker)
```

Ruff (0.12.2)

Local variable `path` is assigned to but never used

Remove assignment to unused variable `path`

(F841)